### ISSUE TITLE
validateStatus doesn't work

### ISSUE THREAD

**Issue by nacimgoura** [3846masa/axios-cookiejar-support#199]
Hello,
with this plugin, `validateStatus` parameter is ignored.
For example (the url called necessarily returns an error 500)
```js
try {
	axios.defaults.validateStatus = () => true;
	const result = await axios.get(`/ws/lieux-vente/RUN/stocks/14`);
	console.log(result); // NOT DISPLAYED WITH THIS PLUGIN
catch (err){
	console.error(error); // DISPLAYED WITH THIS PLUGIN, THIS SHOULD NOT BE THE CASE
}

**Comment by DustinWoods** [comment]
I'm interested in resolving this too.
Here's the culprit: https://github.com/3846masa/axios-cookiejar-support/blob/master/lib/interceptors/request.mjs#L38

Deleting this line doesn't yet produce any issues for me. I'm still trying to figure out the purpose.

**Comment by 3846masa** [comment]
Sorry for the delay in response.
This issue has been resolved with the release of v2.0.0.

If you think this issue is still unresolved, please reopen it.
